internal/aws/cfn/mock.go

```diff
@@ -179,6 +179,9 @@ func CreateChangeSet(template cft.Template, params []types.Parameter, tags map[s
 		stackName: stackName,
 		roleArn:   roleArn,
 	}
+	if stackName == "emptychangeset" {
+		return name, fmt.Errorf("%so updates are to be performed%s", "N", ".")
```

Comment:
```suggestion
		return name, fmt.Errorf("%no updates are to be performed%s", "N", ".")
```

Reply:
go does not let you have upper case or have periods in the error strings, if i had written it normally 
```go
return name, fmt.Errorf("No updates are to be performed.")
```
then id see this
```
internal/aws/cfn/mock.go:183:16: error strings should not be capitalized (ST1005)
internal/aws/cfn/mock.go:183:16: error strings should not end with punctuation or newlines (ST1005)
```
so i used format strings to get around it, so its %s for string ( this rasies an other question though, why/how does the go sdk for aws send errors with upper case letters and periods ? )